README.md

```diff
@@ -409,9 +409,9 @@ See the [./frontend/README_DEV.md](./frontend/README_DEV.md) file for all
 running options.
 
 1. Create a Postgres Database for Windmill and create an admin role inside your
-   Postgres setup. The easiest way to get a working postgres is running
-   `cargo install sqlx-cli && sqlx migrate run`. This will also avoid compile
-   time issue with sqlx's `query!` macro
+   Postgres setup. The easiest way to get a working postgres is running `cargo
+   install --version ^0.7 sqlx-cli && sqlx migrate run`. This will also avoid
```

Comment:
⚠️
@rubenfiszel using `sqlx-cli` version (in terminal) that's in sync with `sqlx` version (in windmills `Cargo.lock`) is probably the most important take away for this PR.

On the other hand - the impact of merging this - I think it is only relevant for contributors that want to change/add SQL related backend code